apps/app/repository.tf

```diff
@@ -40,19 +42,20 @@ resource "github_team_members" "owners" {
 }
 
 resource "github_team_repository" "owners_repo_permissions" {
-  team_id    = github_team.owners.id
-  repository = github_repository.app.name
+  count      = local.create_repo ? 1 : 0
+  team_id    = github_team.owners[0].id
+  repository = github_repository.app[0].name
   permission = "push"
 }
 
 resource "github_repository_file" "codeowners" {
-  repository          = github_repository.app.name
-  branch              = var.environment
+  repository          = github_repository.app[0].name
+  branch              = "main"
```

Comment:
This might be a problem if we have different owners in different environments? I think owners was maybe not the right name either on that front, perhaps we should have a separate block for `reviewers`. Use case being for prod you might add infosec people etc. to the required reviewers but might not want that in staging or dev

Reply:
discussed on slack. will create a backlog item for this